protos/flyteidl/core/workflow.proto

```diff
@@ -194,3 +199,15 @@ message WorkflowTemplate {
     // workflow defaults
     WorkflowMetadataDefaults metadata_defaults = 7;
 }
+
+// Optional task node overrides that will be applied at task execution time.
+message TaskNodeOverrides {
+    // A customizable interface to convey resources requested for a task container. 
+    Resources resources = 1;
```

Comment:
Are we going to implement override for all three fields right now? If we are only planning to implement a subset all the way, then let's remove the others and only add them when we decide to in the future...

Reply:
sure, scoped down to just Resources to begin with